src/handlers/actions/makeArbitraryTransaction.ts

```diff
@@ -52,7 +52,6 @@ export default async (event: ContractEvent): Promise<void> => {
   await writeActionFromEvent(event, colonyAddress, {
     type: ColonyActionType.MakeArbitraryTransaction,
     initiatorAddress: receipt.from,
-    recipientAddress: receipt.to,
```

Comment:
I see what you mean now, it makes sense to remove this as it would always point to the colony rather than the target contract address of the arbitrary transaction.